src/CircuitBreaker.ts

```diff
@@ -62,12 +68,17 @@ export class CircuitBreaker extends EventEmitter {
     }
 
     async _evaluate() {
-        this._evaluatingPromiseHook = Promise.allSettled(this._probes.map(p => p.check()));
+        this._promError = false;
+        this._evaluatingPromiseHook = Promise.allSettled(this._probes.map(p => p.check().catch(error => {
+            if (error instanceof Error && error.message === 'Failed to query Prometheus') {
```

Comment:
why checking this specific kind of error?

whatever the error, it means the breaker is not working as expected

Reply:
https://github.com/scality/breakbeat/pull/11/files#diff-359e4042f12fa2b33c080ef07d2ff300cdd9137a3f55c0dc89b5d823527da29fR71-R75